airflow/providers/apache/hive/operators/hive.py

```diff
@@ -95,8 +95,11 @@ def __init__(
         self.mapred_queue = mapred_queue
         self.mapred_queue_priority = mapred_queue_priority
         self.mapred_job_name = mapred_job_name
-        self.mapred_job_name_template = conf.get('hive',
-                                                 'mapred_job_name_template')
+        self.mapred_job_name_template = conf.get(
+            'hive', 'mapred_job_name_template', fallback='')
+        if self.mapred_job_name_template == '':
+            self.mapred_job_name_template = "Airflow HiveOperator task for " \
+                                            "{hostname}.{dag_id}.{task_id}.{execution_date}"
```

Comment:
```suggestion
        self.mapred_job_name_template = conf.get(
            'hive', 'mapred_job_name_template', fallback="Airflow HiveOperator task for " \
                                            "{hostname}.{dag_id}.{task_id}.{execution_date}")
```

(I've messed up the formatting, but I think you get the idea)

Reply:
OK. I think I've spent far too much time on this one just because it is screwed up. Yeah. I added ~ in the default value and it seems to work fine with fallback now in all combinations. The problem is that it works differently in tests and when you simply import the operator - depending on the configuration of this variable, the imports works or not ... :shrug: